### ISSUE TITLE
Historical entries not appearing in auto-completion

### ISSUE THREAD

**Issue by rdgoite** [vimperator/vimperator-labs#600]
##### Issue type:
- Bug report

##### Version:

```
Vimperator:	3.14.0 (created: 2016/06/07 23:58:16)
Firefox:	Mozilla/5.0 (Macintosh; Intel Mac OS X 10.11; rv:49.0) Gecko/20100101 Firefox/49.0
```

##### Description:

I recently updated to Firefox 49.0, and now no historical entries appear in my auto-completion view.

##### Steps to reproduce:

1. Call `:tabopen`.
1. Try searching for any known URLs previously visited.

**Comment by wilsoniya** [comment]
Can confirm reproducability of this issue on Linux. 

Vimperator: 3.14.0
Firefox: Mozilla/5.0 (X11; Linux x86_64; rv:49.0) Gecko/20100101 Firefox/49.0

**Comment by timss** [comment]
Thanks for the bug report.

Closing as duplicate of #590. Fix merged in #591. Temporary workaround for 3.14 [here](https://github.com/vimperator/vimperator-labs/issues/590#issuecomment-248833125).
